### DISCUSSION TITLE
Does SSIM support 3D images ?

### DISCUSSION THREAD

**Discussion by venkatesh-thiru** [francois-rozet/piqa#4] [Q&A] [answered]
does the SSIM and MS-SSIM support 3D(N,C,D,H,W) images?

**Comment by francois-rozet** [comment, marked as answer]
Hello @v3nkyc0d3z 👋 No it doesn't currently, but I can very easily add the feature. Could you please fill in a "Feature Request" such that I know exactly what is needed ?

Edit: I've made a PR (#5) and merged it. Now SSIM supports 3D images!
